src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        super.setType("Circle");
+        super.setColor(color);
+    }

```

Comment:
```suggestion
    public Circle(int radius, String color, String type) {
        super(type, color)
        this.radius = radius;
        super.setType("Circle");
        super.setColor(color);
    }

```

let's create constructor in Figure class and call it from child classes